injector/README.md

```diff
@@ -1,11 +1,27 @@
 # Injector for the giuroll DLL
+Applies Giuroll to Hisoutensoku without requiring `SWRSToys`. Works akin to the legacy `SokurollLoader.exe` used in older copies of the game.
 
 ## Compilation
-compile with `cargo` using one of the `_-686-pc-windows-msvc` toolchains.
+1. Navigate to this directory in the command prompt
+```
+cd <...>/giuroll/injector
```

Comment:
It's for newer contributors like myself. I'm sure it would be very annoying to get DMs all the time from them because they didn't think they should `cd` into the injector.